docs/package.json

```diff
@@ -17,11 +17,11 @@
   "homepage": "https://github.com/KILTprotocol/sdk-js#readme",
   "dependencies": {
     "@kiltprotocol/sdk-js": "workspace:*",
-    "@polkadot/keyring": "^10.0.0",
+    "@polkadot/keyring": "^10.2.1",
```

Comment:
all 9.10.2 versions of the api packages require at least 10.2.1 of the utils packages